frontend/dockerfile/dockerfile2llb/convert.go

```diff
@@ -815,6 +815,7 @@ func toDispatchState(ctx context.Context, dt []byte, opt ConvertOpt) (*dispatchS
 			target.image.OSFeatures = append([]string{}, platformOpt.targetPlatform.OSFeatures...)
 		}
 	}
+	target.image.Platform = platforms.Normalize(target.image.Platform)
```

Comment:
Can we have a test?

Reply:
This expects a base image not created by buildkit, so not easily.